apps/condo/domains/notification/adapters/hcmAdapter.js

```diff
@@ -82,7 +86,7 @@ class HCMAdapter {
 
         CONFIG_VALIDATED_FIELDS.forEach(
```

Comment:
Why not ajv? (just interest 😄 )

Reply:
This code is ported from some old library and left here as is.